src/main/scala/chisel3/util/experimental/decode/TruthTable.scala

```diff
@@ -14,10 +14,10 @@ final class TruthTable(val table: Map[BitPat, BitPat], val default: BitPat) {
     def writeRow(map: (BitPat, BitPat)): String =
       s"${map._1.rawString}->${map._2.rawString}"
 
-    (table.map(writeRow) ++ Seq(s"${" "*(inputWidth + 2)}${default.rawString}")).toSeq.sorted.mkString("\n")
+    (table.map(writeRow) ++ Seq(s"${" "*(inputWidth + 2)}${default.rawString}")).mkString("\n")
   }
 
-  def copy(table: Map[BitPat, BitPat] = this.table, default: BitPat = this.default) = new TruthTable(table, default)
+  def copy(table: Seq[(BitPat, BitPat)] = this.table, default: BitPat = this.default) = new TruthTable(table, default)
```

Comment:
This method is an easy way to sneak in an unsorted `table`, should we sort here?

Reply:
nice, I'm thinking passing the sort option to internal state in `TurhtTable`

Reply:
Do we need a copy method? Is it so bad to just make a new one via the factory method?

Reply:
This is used in QMC, since QMC need copy from:
https://github.com/chipsalliance/chisel3/blob/2b0bc0ecbc9c53882e2104ecd1e1387039be27f3/src/main/scala/chisel3/util/experimental/decode/QMCMinimizer.scala#L294-L306

I wonder should we expose this to user? currently I didn't come up with a user case.